### DISCUSSION TITLE
HX711 SIC or DAT not available

### DISCUSSION THREAD

**Discussion by BruceParmele** [tasmota/tasmotizer#143] [General]
I set up a keg monitoring system using load cells and an HX711 amp.  It worked fine until I shorted it out.  So I built a new system with the new components. But now I am unable to locate the HX711 options for under GENERIC(18) or any other module for that matter.  I am lost.  Has HX711 been removed from the bin? Any help is most appreciated!  
Thanks

**Comment by jziolkowski** [comment]
Hi. You should ask this question in arendst/tasmota repo. This repo is only for the tool to upload firmware.

**Comment by BruceParmele** [comment]
Thank you for your response and the clarification.:)

<http://www.avg.com/email-signature?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
Virus-free.www.avg.com
<http://www.avg.com/email-signature?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>
